automation/Dockerfile_package-based-hostext.template

```diff
@@ -0,0 +1,34 @@
+ARG NAMESPACE="resin"
+FROM ${NAMESPACE}/balena-push-env:latest
+
+RUN apt-get update && apt-get install -y git make automake gcc libtool libtool-bin pkg-config libarchive-dev libcurl4-openssl-dev libssl-dev libgpgme11-dev && rm -rf /var/lib/apt/lists/*
+
+RUN git clone git://git.yoctoproject.org/opkg opkg
+RUN cd ./opkg && ./autogen.sh && ./configure --enable-libsolv  && make && make install
+
+RUN LD_LIBRARY_PATH=/usr/local/lib && ldconfig
+
+RUN mkdir -p /var/lib/opkg /etc/opkg/
+RUN echo 'dest hostapp /hostapp\n\
+option info_dir /var/lib/opkg/info\n\
+option lists_dir /var/lib/opkg/lists\n\
+option status_file /var/lib/opkg/status'\
+> /etc/opkg/opkg.conf
+
+RUN echo 'ARG NAMESPACE=resin\n\
+FROM ${NAMESPACE}/%%BALENA_MACHINE_NAME%%-package-based-hostext:latest AS builder\n\
+ARG PACKAGES="list of packages to force install without dependencies"\n\
+ARG ARCH_LIST="all aarch64 armv8a_tegra jetson_tx2"\n\
```

Comment:
The above ARCH_LIST needs cleaning

Reply:
Yes, this is something I actually forgot to finish, thanks. They should  be generated from the package index. I'll look into it, thanks.

Reply:
Done.